packages/yuudachi/src/deploy.ts

```diff
@@ -26,14 +27,18 @@ import {
 	RefreshScamlistCommand,
 	SponsorCommand,
 	RepostCommand,
+	ReportCommand,
 
 	// Context Menu
 	HistoryUserContextCommand,
 	SponsorUserContextCommand,
 	RepostMessageContextCommand,
 	ClearContextCommand,
+	ReportMessageContextCommand,
 } from './interactions/index.js';
 
+inspect.defaultOptions.depth = 5;
```

Comment:
Don't know if I should keep this, it's useful to visualize issues

Reply:
Will remove it

Reply:
This can be removed.